src/app.rs

```diff
@@ -10,12 +9,8 @@ use crate::{Error, Middleware};
 /// The default value of the maximum number of concurrent connections.
 const DEFAULT_MAX_CONNECTIONS: usize = 256;
 
-/// The default value of the response timeout in seconds.
-const DEFAULT_RESPONSE_TIMEOUT: u64 = 60;
-
 pub struct App<State> {
     max_connections: usize,
-    response_timeout: Duration,
```

Comment:
I plan on reintroducing this functionality as a built-in middleware exported from the middleware mod.